README.md

```diff
@@ -1,15 +1,21 @@
 # jupyterlab_spellchecker
+![Github Actions Status](https://github.com/jupyterlab-contrib/jupyterlab_spellchecker/workflows/Build/badge.svg)
+[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/jupyterlab-contrib/jupyterlab_spellchecker/master?urlpath=lab)
 
 A JupyterLab extension highlighting misspelled words in markdown cells within notebooks and in the text files.
 
 ![](demo.gif)
 
 The JupyterLab extension is based on [the spellchecker Jupyter Notebook extension](https://github.com/ipython-contrib/jupyter_contrib_nbextensions/tree/master/src/jupyter_contrib_nbextensions/nbextensions/spellchecker) and relies on [Typo.js](https://github.com/cfinke/Typo.js) for the actual spell checking. Spellchecker suggestions are available from the context menu.
 
-The extension provides (Hunspell) [SCOWL](http://wordlist.aspell.net/) dictionaries for American, British, Canadian, and Australian English.
+The extension provides (Hunspell) [SCOWL](http://wordlist.aspell.net/) dictionaries for:
+- American, British, Canadian, and Australian English
+- French,
+- Portuguese,
+- Spanish.
```

Comment:
- German (Germany, Austria, Switzerland)

is missing in README.

Reply:
Good catch, thanks!